stimela/__init__.py

```diff
@@ -92,7 +93,10 @@ def _is_from_subprocess(rec):
         log_formatter = log_boring_formatter if boring else log_colourful_formatter
 
         if console:
-            log_console_handler = MultiplexingHandler()
+            if "SILENT_STDERR" in os.environ and os.environ["SILENT_STDERR"].upper()=="ON":
```

Comment:
I prefer ``os.environ.get("SILENT_STDERR", "").upper() == "ON"`` but that's just coding fancy.